src/redux/books/booksSlice.js

```diff
@@ -1,30 +1,74 @@
 import {
   createSlice,
+  createAsyncThunk,
 } from '@reduxjs/toolkit';
-import books from '../../components/books';
 
-const initialState = {
-  books,
-};
+const booksURL = 'https://us-central1-bookstore-api-e63c8.cloudfunctions.net/bookstoreApi/apps/YTkkGNjFzdVaLfwvsjLv/books';
+const Bookss = [];
 
-const booksSlice = createSlice({
-  name: 'book',
-  initialState,
-  reducers: {
-    addBook: (state, action) => {
-      state.books.push(action.payload);
-    },
-    removeBook: (state, action) => ({
-      ...state,
-      books: state.books.filter((book) => book.id !== action.payload),
+export const booksDisplay = createAsyncThunk(
+  'bookstore/books',
+  async () => {
+    const response = await fetch(booksURL);
+    const data = await response.json();
+    Object.keys(data).forEach((ind) => {
+      const each = {
+        id: ind,
+        title: data[ind].title,
+        author: data[ind].author,
+        category: data[ind].category,
+      };
+      Bookss.push(each);
+    });
+    return Bookss;
+  },
+);
+
+export const booksAdd = createAsyncThunk('bookstore/books/ADD_BOOK', async (book) => {
+  await fetch(booksURL, {
+    method: 'POST',
+    headers: { 'Content-Type': 'application/json' },
+    body: JSON.stringify({
+      item_id: book.id,
+      title: book.title,
+      author: book.author,
+      category: 'Fiction',
```

Comment:
- [ ] I am able to create the book as I can see in the network tab. But unable to see it in the UI. Please fix that.